www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,489 @@ license: >
+
+  Example:
+  ```
+  <dependency id="com.plugin.id" url="https://github.com/myuser/someplugin" commit="428931ada3891801" subdir="some/path/here" />
+  ```
+
+### platform
+
+  Identifies platforms that have associated native code or require modifications to their configuration files. Tools using this specification can identify supported platforms and install the code into Cordova projects. Plugins without `<platform>` tags are assumed to be JavaScript-only, and therefore installable on any and all platforms.
+
+  Attributes(type) | Description
+  ---------------- | ------------
+  name(string) | *Required* <br/> Allowed values: ios, android, blackberry10, amazon-fireos, wp8, windows <br/> Identifies a platform as supported, associating the element's children with that platform.
+
+  Example:
+  ```
+  <platform name="android">
+    <!-- android-specific elements -->
+  </platform>
+  ```
+
+#### source-file
+
+  Identifies executable source code that should be installed into a project.
+
+  Attributes(type) | Description
+  ---------------- | ------------
+  src(string) | *Required* <br/> Location of the file relative to plugin.xml. If the src file can't be found, plugman stops and reverses the installation, issues a notification about the problem, and exits with a non-zero code.
+  target-dir(string) | A directory into which the files should be copied, relative to the root of the Cordova project. In practice, this is most important for Java-based platforms, where a file in the `com.alunny.foo` package must be located within the `com/alunny/foo` directory. For platforms where the source directory is not important, this attribute should be omitted.
+  framework(boolean) | *Default: false* <br/> ==iOS== <br/> If set to true, also adds the specified file as a framework to the project.
+  compiler-flags(string) | ==iOS== <br/> If set, assigns the specified compiler flags for the particular source file.
+
+  Examples:
+  ```
+  <!-- android -->
+  <source-file src="src/android/Foo.java" target-dir="src/com/alunny/foo" />
+  <!-- ios -->
+  <source-file src="src/ios/CDVFoo.m" />
+  <source-file src="src/ios/someLib.a" framework="true" />
+  <source-file src="src/ios/someLib.a" compiler-flags="-fno-objc-arc" />
+  ```
+
+#### header-file
+
+  This is like `<source-file>` element, but specifically for platforms such as iOS and android that distinguish between source files, headers, and resources. This is not supported by Windows. 
+
+  Example:
+
+  For iOS:
+  ```
+  <header-file src="CDVFoo.h" />
+  ```
+
+#### resource-file
```

Comment:
I will add it in the description that it inherits all the attributes of source-file along with these additional attributes.